sources/packages/web/src/components/common/students/StudentApplications.vue

```diff
@@ -8,76 +8,51 @@
       ></body-header>
     </template>
     <content-group>
-      <DataTable
-        :value="applicationsAndCount.results"
-        :lazy="true"
-        :paginator="true"
-        :rows="DEFAULT_PAGE_LIMIT"
-        :rowsPerPageOptions="PAGINATION_LIST"
-        :totalRecords="applicationsAndCount.count"
-        @page="paginationAndSortEvent($event)"
-        @sort="paginationAndSortEvent($event)"
-        :loading="loading"
+      <toggle-content
+        :toggled="!applicationsAndCount.results?.length"
+        message="No applications are currently available."
       >
-        <template #empty>
-          <p class="text-center font-weight-bold">No records found.</p>
-        </template>
-        <Column
-          :field="StudentApplicationFields.ApplicationNumber"
-          :sortable="true"
-          header="Application #"
+        <v-data-table
```

Comment:
This is a lazy pagination. So we must use `v-data-table-server` rather instead of `v-data-table`. 

Please follow the reference code from here. https://github.com/bcgov/SIMS/blob/main/sources/packages/web/src/views/aest/CASInvoices.vue

Reply:
@dheepak-aot I did not see this as lazy because a student may have a few applications over his lifetime. I would recommend helping it as it is right now and saving the server-side pagination for later improvement.

Reply:
As pointed out by @dheepak-aot API is already accepting the pagination arguments (I missed that), hence can change it.